public/js/term-highlight.js

```diff
@@ -0,0 +1,34 @@
+jQuery(function($) {
+  $(document).ready(function () {
+    var searchTerm = $('h1.sr-only').data('term').toLowerCase();
+        // remove speach mark
+        searchTerm = searchTerm.replace(/[".]/g,'');
+        // remove special characters
+        searchTerm = searchTerm.replace(/[;]/g,'');
+        // Change Doctor to Dr
+        searchTerm = searchTerm.replace('doctor','dr');
+
+    var terms = searchTerm.split(/\W+/);
+
+    for (var i = 0, len = terms.length; i < len-1; i++) {
+      terms.push(terms[i]+' '+terms[i+1]);
+    }
+
+    if ($.inArray( searchTerm, terms ) === -1){
```

Comment:
I think ES6 transpiler is the better way of doing this via`babel`, we've partially done this in the JS version to maintain compatibility. I'm asking if this PR is the place to do this or maybe should be part of the front-end tidying?